guessing-game-assignment-mrbgit.html

```diff
@@ -4,29 +4,62 @@
 	counter,
 	secondGuess,
 	thirdGuess,
-	counter;
+	counter,
+	lookup_signer,
+	play_again;
 signers = ["John Adams", "Samuel Adams", "Josiah Bartlett", "Carter Braxton", "Charles Carroll", "Samuel Chase", "Abraham Clark", "George Clymer", "William Ellery", "William Floyd", "Benjamin Franklin","Elbridge Gerry","Button Gwinnett","Lyman Hall","John Hancock","Benjamin Harrison","John Hart","Joseph Hewes","Thomas Heyward, Jr.","William Hooper","Stephen Hopkins","Francis Hopkinson","Samuel Huntington","Thomas Jefferson","Francis Lightfoot Lee","Richard Henry Lee","Francis Lewis","Philip Livingston","Thomas Lynch, Jr.","Thomas McKean","Arthur Middleton","Lewis Morris","Robert Morris","John Morton","Thomas Nelson, Jr.","William Paca","Robert Treat Paine","John Penn","George Read","Caesar Rodney","George Ross","Benjamin Rush","Edward Rutledge","Roger Sherman","James Smith","Richard Stockton","Thomas Stone","George Taylor","Charles Thomson","Matthew Thornton","George Walton","William Whipple","William Williams","James Wilson","John Witherspoon","Oliver Wolcott","George Wythe"];
 counter = 1;
-guessSigners = prompt("Can you name one of the signers of the Decaration? You have 3 guesses.");
-while(counter < 4){
-	if(signers.indexOf(guessSigners) !== -1){
-		alert("Great Job!");
-	}else{
-		secondGuess = prompt("try again");
-		counter++;
-		if(signers.indexOf(secondGuess) !== -1){
-			alert("Second time's a charm!");
-			counter++;
-		}else{
-			thirdGuess = prompt("Last chance!");
+
+function wrong_message (){ //this presents the user with the error message
+	var message
+	if (counter == 1){
+		message = "try again";
+	}
+	else if (counter == 2){
+		message = "Second time's a charm!";
+	}
+	else if (counter == 3){
+		message = "Last chance!";
+	}
+		return message;
+};
+
+function one_wrong_answer(){
+	while((signers.indexOf(guessSigners) < 0) == true){
+	if (counter == 4){ //when there are three tries the game stops
+		alert("Not your day, refresh your browser and try again")
+		(signers.indexOf(guessSigners) < 0) == false;
+	}
+		else if (counter < 4){ //messages for the wrong answers
+			guessSigners = prompt(wrong_message());
 			counter++;
-			if(signers.indexOf(thirdGuess) !== -1){
-				alert("You got it!");
-			}else{
-				alert("Not your day, refresh your browser and try again");
-				counter++;
+			if (signers.indexOf(guessSigners) >= 0){//the correct answer during one of the times finishes the game
+			(signers.indexOf(guessSigners) < 0) == false;
+			play_again = confirm("Great Job! Would you like to play again?");
 			}
 		}
-	}
+	};
+}
+
+function play_game(){
```

Comment:
return true or false from play_game() so you can do something like 
`while (play_game())`